bears/vcs/git/GitCommitBear.py

```diff
@@ -105,8 +105,12 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
         stdout = stdout.rstrip('\n')
         pos = stdout.find('\n')
         shortlog = stdout[:pos] if pos != -1 else stdout
+        if pos != -1:
+            if stdout[pos+2] == '\n':
+                yield Result(self, 'Extra Newlines found.')
         body = stdout[pos+1:] if pos != -1 else ''
```

Comment:
So, you'll still consider the body just after the shortlog part ? or after filtering the extra newlines after shortlog? ^^